docs/cnv-configuration.md

```diff
@@ -0,0 +1,153 @@
+
+
+  **Note**: If both `spec.selector.labels` and`spec.selector.annotations` are set only namespaces
+  that match both will be selected
+
+  **Note**: AAQ must be configured to allow AAQ cluster quota management
+
+  Refer to the "Configure AAQ Operator" section for more details. 
+
+Example manifest:
+```yaml
+apiVersion: aaq.kubevirt.io/v1alpha1
+kind: ApplicationAwareClusterResourceQuota
+metadata:
+  name: example-resource-quota
+spec:
+  quota:
+    hard:
+    requests.memory: 1Gi
+    limits.memory: 1Gi
+    requests.cpu/vmi: "1"
+    requests.memory/vmi: 1Gi
+  selector:
+    annotations: null
+    labels:
+      matchLabels:
+        kubernetes.io/metadata.name: default
+```
+
+### How it works
+
+These new quotas are controller-based rather than admission-based. Here’s how they work:
+
+1. [Scheduling Gates](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness/): a scheduling gates is added to    the created pod via a mutating webhook.
+   See the "Configure AAQ Operator" section below for more information
+2. Controller Evaluation: a controller evaluates whether
+   there is sufficient room for the pod according to the quota.
+3. Quotas Update: If there is enough room, the scheduling gate is 
+   removed from the pod, and the quotas used status is updated accordingly.
+   If not, an event is created explaining why the quota doesn't allow the pod's creation.
+
+For non-compute resources, the quota acts like the standard ResourceQuota.
+
+***Warning***: Pods that set `.spec.nodeName` are not allowed to use in any namespace where AAQ Operates.
+
+non-schedulable resources are supported by both AAQ quota and AAQ cluster 
+quota by managing a k8s resource quota behind the scenes.
+
+The primary purpose of these managed quotas is to set unified quotas for bot native and extended 
+accounting resources, eliminating the need to manage both ResourceQuota and AAQ quota, or
+ClusterResourceQuota and AAQ cluster quota separately.
+
+### Configure AAQ Operator
+To enable the AAQ feature, set the `hco.spec.featureGates.enableApplicationAwareQuota` field to `true`. 
+
+You can do this from the command line with the following command:
+```bash
+$ oc patch hco kubevirt-hyperconverged -n openshift-cnv \
+ --type json -p '[{"op": "add", "path": "/spec/featureGates/enableApplicationAwareQuota", "value": true}]'
+```
+
+#### Configuring AAQ Settings
+The applicationAwareConfig object within the HyperConverged resource's spec allows you to customize AAQ behavior. Key fields include:
+* `vmiCalcConfigName` -  Defines how resource counting is managed for VM-associated pods. The available options are:
+   * `VmiPodUsage` - Counts compute resources for VM-associated pods in the same way as native resource quotas, while excluding migration-related resources.
+   * `VirtualResources` - Counts compute resources based on the VM's specifications, using the VM's RAM size for memory and CPU threads for processing.
```

Comment:
I think its better to say virtual CPUs (vCPU) rather then using the term threads. Perhaps its also better to elaborate on the way you calculate the vCPU

Reply:
Done